cmd/dvotenode/dvotenode.go

```diff
@@ -71,8 +71,8 @@ func newConfig() (*config.DvoteCfg, config.Error) {
 		"use developer mode (less security)")
 	globalCfg.PprofPort = *flag.Int("pprof", 0,
 		"pprof port for runtime profiling data (zero is disabled)")
-	globalCfg.LogLevel = *flag.StringP("logLevel", "l", "info",
-		"log level (debug, info, warn, error, fatal)")
+	globalCfg.LogLevel = *flag.StringSliceP("logLevel", "l", []string{"*:info"},
+		"log level for each subsystem, in format <subsystem|*>:<debug|info|warn|error|fatal>")
```

Comment:
I would say make the default just `info` rather than `*:info`, then we don't break existing commands.

Reply:
Ah, I see you still support it, but I didn't get that from your docs.

Reply:
i updated the docs to be clearer